public/components/explorer/visualizations/datapanel.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import './datapanel.scss';
+import './field_item.scss';
+import React from 'react';
+import _ from 'lodash';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormControlLayout,
+  EuiSpacer
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { FieldList } from './fieldList';
+
+export const DataPanel = (props: any) => {
+
+  const { schema } = props.queryResults;
+
+  return (
+    <EuiFlexGroup
+      gutterSize="none"
+      className="lnsInnerIndexPatternDataPanel"
+      direction="column"
+      responsive={false}
+    >
+      <EuiFlexItem grow={false}>
+        <EuiFormControlLayout
+          icon="search"
+          fullWidth
+          clear={{
+            title: i18n.translate('xpack.lens.indexPatterns.clearFiltersLabel', {
```

Comment:
i don't think xpack should be here?

Reply:
Same thing here as I was borrowing code from existing kibana lens and only made minimal changes. I'll remove these unrelated stuff.